TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/TrafficReplayer.java

```diff
@@ -272,10 +272,10 @@ public static void main(String[] args) throws Exception {
         try {
             uri = new URI(params.targetUriString);
         } catch (Exception e) {
-            final var msg = "Exception parsing " + params.targetUriString;
-            System.err.println(msg);
-            System.err.println(e.getMessage());
-            log.atError().setMessage(msg).setCause(e).log();
+            log.atError().setMessage("Exception parsing {}")
+            .addArgument(params.targetUriString)
```

Comment:
Is there a styling tool, on this project?  Lets see about using spotless or similar to make sure that indenting is consistent

Reply:
Not currently, that should be a discussion. I've since simplified this change to be only a few changes that are bugs until discussion is finalized for #633